src/lib/ui/ClimateSettingForm/ClimateSettingScheduleFormClimateSetting.tsx

```diff
@@ -56,11 +43,41 @@ export function ClimateSettingScheduleFormClimateSetting({
               <InputLabel>{t.climateSetting}</InputLabel>
               <span className='seam-label'>{t.climateSettingSubHeading}</span>
             </div>
-            <FormContent
-              control={control}
-              resetField={resetField}
-              hvacModeSetting={hvacModeSetting}
-            />
+            <FormField>
+              <Controller
+                name='climateSetting'
+                control={control}
+                render={({ field: { value, onChange } }) => {
+                  return (
+                    <ClimateSettingControlGroup
+                      mode={value.hvacModeSetting}
+                      onModeChange={(mode) => {
+                        onChange({ ...value, hvacModeSetting: mode })
+                      }}
+                      coolValue={value.coolingSetPoint}
+                      heatValue={value.heatingSetPoint}
+                      delta={5}
+                      maxCool={90}
+                      maxHeat={100}
```

Comment:
Where do these default come from? Can these be removed now that this is merged? https://github.com/seamapi/react/commit/ea548fb2d8a8db467bb03a507a043c73445e2997

Reply:
https://github.com/seamapi/react/pull/545/commits/24ceb055c0606202bdb65f94c683eed16b17b5fa#diff-8cc510eb20db8ba8389bec8f58a2cb60786d5736044eae968583dab1e3e03306L59-L63